packages/engine/Specs/Scene/Vector3DTileContentSpec.js

```diff
@@ -613,6 +614,19 @@ describe(
           expect(scene).toRender(whitePixel);
         });
       });
+
+      it("sets the heightReference for Vector3DTilePoints", () => {
+        const heightReference = HeightReference.CLAMP_TO_3D_TILE;
+        return Cesium3DTilesTester.loadTileset(scene, vectorTilePointsTileset, {
```

Comment:
We'd prefer moving any new tests to the `async/await` pattern rather than promise chains. The helps make the code flatter and more readable. I.e.,

```
const tileset = await Cesium3DTilesTester.loadTileset(...);
const vectorTile = tileset._root.children[0];
const vectorTilePoint = vectorTile._content._points;
expect(vectorTilePoint._heightReference).toEqual(heightReference);
```